### ISSUE TITLE
Takk the Leaper (wrong run speed)

### ISSUE THREAD

**Issue by dolphindude278** [elysium-project/bug-tracker#101]
The rare raptor pet Takk the Leaper doesn't have the correct pursuit speed. Takk is suppose to have the fastest run speed of any pet in vanilla.

**Comment by dolphindude278** [comment]
Takk was known for his supersonic pursuit speed(200% movement speed). He could catch 60% movement speed mounts and keep up with 100% speed mounts.

This was removed in patch 1.9. 

Base pet speed is now standardized for all pets, including legacy pets. Pet speed can still be modified through the talent Bestial Swiftness and pet abilities such as Charge, Dash, etc.

https://www.google.com/amp/s/www.engadget.com/amp/2011/08/16/wow-archivist-patch-1-9-the-gates-of-ahnqiraj/

**Comment by dolphindude278** [comment]
This link mentions his 'supersonic chase speed' he was famous for in vanilla 

https://wow.gamepedia.com/Takk_the_Leaper

**Comment by dolphindude278** [comment]
Can't find a link of his exact movement speed but every mention of Takk mentions his speed being the same as an epic mount.

**Comment by Terranzin** [comment]
I´m reviewing your issue and I have a problem finding solid proof. I do find notes/comments about his speed from the early versions of World Of Warcraft (patch 1.2+) up to 1.8 when he got nerfed, such as:

"_I was using eyes of the beast (I think thats how its written) and was running beside a non-mounted PC and passed him like the guys was walking. Takk is awesome and fast!_ "
And
" _but one thing for sure is that he does have 2sec atk speed and runs fast._"

Looked into a older archieved WoWWiki from october 16th 2006 which states:
"_This rare spawn was once popular for his supersonic chase speed, but he was eventually normalized to normal speed_."

It´s also stated this was changed around patch 1.9 and what I found in the patch notes was following
"_base pet speed is now standardized for all pets, including legacy pets_"

Conclusion, I do believe that Takk The leaper (Id: 5842) does in matter of fact does run faster then it is today on Elysium. I´ll set this as confirmed from QA and handing the issue over to our devs.
